CDP4Dal/CDP4Dal.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageTags>CDP COMET ECSS-E-TM-10-25</PackageTags>
     <PackageLicenseExpression>LGPL-3.0-only</PackageLicenseExpression>
     <PackageReleaseNotes>
-      [Update] to CDP4Common 8.1.0
+      [Update] .Net Framework to System.Reactive 4.1.5

```

Comment:
please rephrase to 
[Update] System.Reactive to version 4.1.5
[Drop] support for net45;net451;net452